glide-core/src/protobuf/redis_request.proto

```diff
@@ -234,6 +234,10 @@ enum RequestType {
     Dump = 193;
     Restore = 194;
     SortReadOnly = 195;
+    XPending = 196;
+    XClaim = 197;
+    XAutoClaim = 198;
+    XInfo = 199;
```

Comment:
I still don't undestand how it is related to `xpending` command

Reply:
it is not
I'm just tired of adding these for each command